app/views/devise/registrations/new.html.erb

```diff
@@ -14,11 +14,8 @@
   </div>
 
   <div class="field">
-    <%= f.fields_for :department do |d| %>
-      <%= d.label :department %>
-      <%= d.select :name,
-          Department::DEPARTMENT, class: "department dropdown"%>
-    <% end %>
+      <%= f.label :department_id %>
+      <%= collection_select(:department_id, :department_id, Department.all, :id, :name) %>
```

Comment:
do you know why there are two `:department_id`s here, and what happens when you take one out?  Looking at the docs, the example only has one key